x-pack/test/api_integration/apis/telemetry/telemetry_local.ts

```diff
@@ -106,7 +106,6 @@ export default function ({ getService }: FtrProviderContext) {
       expect(stats.stack_stats.kibana.plugins.saved_objects_counts.others).to.be(0); // Unless there's a bug/unexpected situation, it should be 0
       expect(stats.stack_stats.kibana.plugins.saved_objects_counts.non_registered_types).to.eql([]); // During tests, we shouldn't expect to list types that are not registered.
 
-      expect(stats.stack_stats.kibana.plugins.reporting.enabled).to.be(true);
```

Comment:
Head's up @elastic/kibana-core: Reporting will no longer use snapshot telemetry, hence this removed line.

Reply:
Update, will rethink this. It probably makes sense to keep the `enabled/available` logic in the legacy Reporting telemetry.